extensions/3rdparty/ArrayExtension/ArrayExtension.php

```diff
@@ -519,7 +519,7 @@ function arrayslice( &$parser, $arrayid_new , $arrayid , $offset, $length='') {
 *    {{#arrayintersect:arrayid_new|arrayid1|arrayid2}}
 *   See: http://www.php.net/manual/en/function.array-intersect.php
 */
-    function arrayintersect( &$parser, $arrayid_new , $arrayid1 , $arrayid2 ) {
+    function arrayintersect( $parser, $arrayid_new , $arrayid1 , $arrayid2 ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$parser". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of returns of this function 4, down to the maximum allowed 3. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)